### PR TITLE
Added dockerfile runtime support

### PR DESCRIPTION
**Pull request by mhristof**
Covered by mhristof.rst

### CODE REVIEW

File: doc/en/getting_started.rst [hunk 1]
```diff
@@ -305,6 +305,92 @@ Installation using a `virtualenv <https://virtualenv.pypa.io/en/stable>`_:
...
+
+If your testplan file has a name other than ``test_plan.py``, you can add it as an
+argument in the ``docker run`` command:
+
+    .. code-block:: bash
+
+        docker run -v /home/user/code:/work -it mhristof/testplan:2 ./my_test_plan.py
+
+
+ If you require special arguments for ``test_plan.py``, you can just append them
```
Comment (mhristof) [inline doc/en/getting_started.rst]: there is something rotten here and im not sure what. The indentation level is 1 more than it should be

image

Comment (JohnChiotis) [reply doc/en/getting_started.rst]: I ll fix that along with some other typos.

File: doc/en/getting_started.rst [hunk 2]
```diff
@@ -305,6 +305,92 @@ Installation using a `virtualenv <https://virtualenv.pypa.io/en/stable>`_:
...
+
+Docker images for testplan are provided for two python versions, ``python2`` and
+``python3``.
+
+The images can be retrieved with the following commands:
+
+    .. code-block:: bash
+
+        # Python 2
+        docker pull mhristof/testplan:2
```
Comment (JohnChiotis) [inline doc/en/getting_started.rst]: sudo?

File: doc/en/getting_started.rst [hunk 3]
```diff
@@ -305,6 +305,92 @@ Installation using a `virtualenv <https://virtualenv.pypa.io/en/stable>`_:
...
+
+        # Python 3
+        docker pull mhristof/testplan:3
```
Comment (JohnChiotis) [inline doc/en/getting_started.rst]: sudo?

File: doc/en/getting_started.rst [hunk 4]
```diff
@@ -305,6 +305,92 @@ Installation using a `virtualenv <https://virtualenv.pypa.io/en/stable>`_:
...
+
+
+Interactive mode
+++++++++++++++++
+
+To launch testplan in test mode, you can type:
+
+    .. code-block:: bash
+
+        docker run -it mhristof/testplan:2 bash
```
Comment (JohnChiotis) [inline doc/en/getting_started.rst]: sudo?

File: doc/en/getting_started.rst [hunk 5]
```diff
@@ -305,6 +305,92 @@ Installation using a `virtualenv <https://virtualenv.pypa.io/en/stable>`_:
...
+Batch execution mode
+++++++++++++++++++++
+
+To run testplan docker image in batch mode, you'll need to add your code as a
+docker volume when running the image. If the code to test is in ``/home/user/code``,
+the docker command will be:
+
+    .. code-block:: bash
+
+        docker run -v /home/user/code:/work -it mhristof/testplan:2
```
Comment (JohnChiotis) [inline doc/en/getting_started.rst]: sudo?

File: doc/en/getting_started.rst [hunk 6]
```diff
@@ -305,6 +305,92 @@ Installation using a `virtualenv <https://virtualenv.pypa.io/en/stable>`_:
...
+
+
+If your testplan file has a name other than ``test_plan.py``, you can add it as an
+argument in the ``docker run`` command:
+
+    .. code-block:: bash
+
+        docker run -v /home/user/code:/work -it mhristof/testplan:2 ./my_test_plan.py
```
Comment (JohnChiotis) [inline doc/en/getting_started.rst]: ditto

**Pull request merged**